Makefile

```diff
@@ -7,11 +7,12 @@ timed:
 	time raco make main.rkt
 
 test: all
+	# cd src/backend-c/runtime/tests && make all && ./tests
 	raco test .
 
 clean:
 	rm -f *~ *#* *.c *.out *.o *.s
 	find . -path '*/compiled/*' -delete
 	find . -type d -name "compiled" -delete
-	cd src/backend-c/runtime/; make clean
+	cd src/backend-c/runtime/; make clean; cd tests && make clean
```

Comment:
Could you add this to the "clean" recipe for `src/backend-c/runtime/Makefile`? I think that would be a more useful place for this.

Reply:
Done.